README.rst

```diff
@@ -1,21 +1,27 @@
 ==========================
```

Comment:
@dcoa, I don't think we are mentioning anything about the [`AuditNote`](https://github.com/eduNEXT/eox-audit-model/blob/287ec3f4b5ab63185cdb11e54a4752d4dd8e02f2/eox_audit_model/models.py#L188) model in this readme. We should check if there is anything worth mentioning from there.